lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -500,6 +500,7 @@ enum LoopControlMask {
 enum ExtendedControls {
     InitiationIntervalINTEL = 5889,
     MaxConcurrencyINTEL = 5890,
+    DependencyArrayINTEL = 5891,
```

Comment:
According to the latest version of the spec there is no extended controls anymore
https://github.com/KhronosGroup/SPIRV-Registry/commit/ce3e08be38bdec76a60f4e4a3bef36a1fd2ca51c#diff-2d7da671b991b8c0fd2cde6707831bab

Reply:
@AlexeySotkin How would you rather I resolved this? Should I include a spec-aligning commit into this PR, or would leaving a `TODO` and tackling it with a new PR suffice?

Reply:
I'd prefer to align with the spec in scope of a separate PR.

Reply:
OK, I've added a TODO for this.